pages/Transaction.tsx

```diff
@@ -204,40 +204,30 @@ function TransactionDetailRow(props: {
 }
 
 function PodcastingInfo({ boost }: { boost: Boostagram }) {
+  const renderDetail = (title: string, content: any) => {
+    if (content != null && content !== "") {
```

Comment:
`!= null` also checks for undefined so it will be. Did this because the error was happening due to ts being 0 (which is a valid value as the boostagram has been sent before the podcast began)

Reply:
It just does 😄 https://stackoverflow.com/a/15992131

Reply:
ah because of `!=` and not `!==` 👍 
maybe can you add a comment. somebody else might go there and change it to `!==` without thinking about it.

Reply:
Ah got it now, cool will add a comment :+1:

Reply:
What about `content === 0 || !!content`? either 0 or truthy (non-empty value)

Reply:
That's best, changed!